test/verify/check-system-info

```diff
@@ -895,6 +895,36 @@ password=foobar
         b.wait_in_text(".pf-c-menu__item.pf-m-selected", "Custom crypto policy")
         b.click("#crypto-policy-dialog button.pf-c-button.pf-m-link")
 
+    @skipImage("crypto-policies not avaiable", "fedora-coreos", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-stable", "arch")
+    def testFIPSPolicy(self):
```

Comment:
This specifically tests the "clean up broken state" case, so perhaps `testInconsistentFIPSPolicy`?

Reply:
Or should it start with enabling FIPS the supported way, then putting the VM in an incosistent state and then testing that?

Reply:
That works as well, sure!